src/sig_stfl/sig_stfl.h

```diff
@@ -291,7 +294,7 @@ typedef struct OQS_SIG_STFL_SECRET_KEY {
 	/* mutual exclusion struct */
 	void *mutex;
 
-	/* file storage handle */
+	/* Application managed data related to secure storage of secret key data */
```

Comment:
How does the application have to handle this? Does it have to allocate/deallocate? Where does it pass this context in? Again, what about an example (similar to "example_kem/_sig")?

Reply:
Added an example app example_sig_stfl.